docs/src/joint/api/dia/Paper/events.html

```diff
@@ -308,6 +308,38 @@
       </table>
     </td>
   </tr>
+  <tr>
+    <th>pan</th>
+    <td>
+      <p>Triggered when a pan event is emitted while pointer is on top the target. Pan events are simillar to mousewheel events, but while mousewheel events cover only on the Y axis, pan events cover both X and Y axis. This is usually possible with touchpads / trackpad devices.</p>
```

Comment:
"...while **the** pointer is on top **of** the target."

"similar" (spelling)

"...but while mousewheel events cover only the Y axis..." (delete "on")